### ISSUE TITLE
Organic keywords

### ISSUE THREAD

**Issue by adriaanvanrossum** [simpleanalytics/roadmap#116]
Display which keywords were used to find the webpage.

**Comment by adriaanvanrossum** [comment]
How to do this with Google: https://github.com/googleapis/google-api-nodejs-client/blob/master/samples/webmasters/query.js

**Comment by adriaanvanrossum** [comment]
Our next big feature will be this!

**Comment by Macro-Jackson** [comment]
Yes! Yes! Yes, please. It is very important to see what keywords we are found on (and where not).

**Comment by adriaanvanrossum** [comment]
Do note that this will require Google Search Console API to connect with Simple Analytics. We can't get this data otherwise. So basically it's another view of google data.

**Comment by jnardiello** [comment]
So, this is a very useful info - BUT what's the catch with google data? I'm not sure I get the implications of this.

**Comment by adriaanvanrossum** [comment]
The catch is that you still need to get it from Google. So you need to allow read access to Simple Analytics to your Google Search Console account. I don't think there are many implications.

**Comment by jnardiello** [comment]
Right - it's a core feature imho. Thanks for the immediate response.

**Comment by alinacsava** [comment]
A great feature to have!

But please also add the option to hide keywords from public stats. For example my site https://simpleanalytics.com/fontsarena.com for which I hide Top Pages, I want my stats to be open, but being a content website it's understandable I don't want to make public things like Google keywords.

**Comment by adriaanvanrossum** [comment]
That makes sense Alina!

**Comment by Macro-Jackson** [comment]
Any updates here?

**Comment by mbardelmeijer** [comment]
Any updates on this? 😄

**Comment by askerrit** [comment]
👋
